src/ircutils.py

```diff
@@ -345,6 +345,25 @@ def banmask(hostmask):
         else:
             return '*!*@'  + host
 
+
+def accountExtban(nick, irc):
```

Comment:
nit: having `irc` as the first argument is more consistent with the other functions here

Reply:
Right. I probably wanted to make it optional or something, but I don't think it makes sense.